doc/api/http.md

```diff
@@ -695,6 +695,10 @@ Once a socket is assigned to this request and is connected
 ### request.setTimeout(timeout[, callback])
 <!-- YAML
 added: v0.5.9
+changes:
+  - version: v9.0.0
+    pr-url: https://github.com/nodejs/node/pull/8895
+    description: consistently set socket timeout only when the socket connects
```

Comment:
For consistency with other descriptions:
```suggestion
    description: Consistently set socket timeout only when the socket connects.
```

Reply:
Done, thanks! :)